tests/unit/test_jenkins.py

```diff
@@ -1506,3 +1506,65 @@ def test_remove_unlisted_plugins(  # pylint: disable=too-many-arguments
         )
     else:
         mock_client.delete_plugins.assert_not_called()
+
+
+def test__invalidate_sessions(
+    mock_client: unittest.mock.MagicMock, container: ops.Container, monkeypatch: pytest.MonkeyPatch
+):
+    """
+    arrange: given a monkeypatched client that runs a groovy script without errors.
+    act: when _invalidate_sessions is called.
+    assert: no errors are raised.
+    """
+    mock_client.run_groovy_script = lambda *_args, **_kwargs: None
+    monkeypatch.setattr(jenkins, "_get_client", lambda *_args, **_kwargs: mock_client)
+
+    jenkins._invalidate_sessions(container)
+
+
+def test__set_new_password(
```

Comment:
Same here

Reply:
This was to increase coverage as well, i could opt in for just skipping these lines.